Gemfile

```
@@ -6,10 +6,13 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 ruby '2.6.5'
 
 gem 'bootsnap', '>= 1.4.2', require: false
+gem 'config'
+gem 'http'
```

```suggestion
gem 'http', '~> 4.0'
```

Let's peg to a version. That ok?

---

Same conversation for `warden` and `config` too of course